src/api/lib/data.ts

```diff
@@ -2,7 +2,8 @@ import fs from 'fs'
 import { DB, ServiceKey } from '../types/admin'
 
 const data: DB = {
-  serviceKeys: []
+  serviceKeys: [],
+  domains: []
```

Comment:
you are storing the `mapping` resource, so `mappings` make more sense here.

Reply:
Could you create a helper function called `getMappings` here? so other developers can call the `getMappings` function.